OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -950,12 +950,9 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 				self.QueueActivity(order.Queued, WrapMove(new Move(self, cell, WDist.FromCells(8), null, true, Color.Green)));
 				self.ShowTargetLines();
 			}
-
-			// TODO: This should only cancel activities queued by this trait
-			if (order.OrderString == "Stop")
-				self.CancelActivity();
-
-			if (order.OrderString == "Scatter")
+			else if (order.OrderString == "Stop")
+				self.CancelActivity(); // TODO: This should only cancel activities queued by this trait
```

Comment:
The comment here can still go on the line above the `else if` without causing an error and again adhering to the Coding Standard.